wait/all_test.go

```diff
@@ -0,0 +1,97 @@
+package wait
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"io"
+	"testing"
+	"time"
+)
+
+func TestMultiStrategy_WaitUntilReady(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		ctx    context.Context
+		target StrategyTarget
+	}
+	tests := []struct {
+		name     string
+		strategy Strategy
+		args     args
+		wantErr  bool
+	}{
+		{
```

Comment:
Could you add a few test cases where `wantErr` is true? 🙏

Reply:
resolved via [adfb44b](https://github.com/testcontainers/testcontainers-go/pull/580/commits/adfb44b9f079b1e7c5691988cc3fdc0927385199)